packages/toolkit/utils/package.json

```diff
@@ -50,7 +50,6 @@
       "import": "./dist/esm/universal/pluginDagSort.js",
       "default": "./dist/cjs/universal/pluginDagSort.js"
     },
-    "./ajv": "./compiled/ajv/index.js",
```

Comment:
We can delete the `compiled/ajv` and `compiled/ajv-keywords` folder now

Reply:
And` better-ajv-errors`

Reply:
done!